posthog/models/proxy_record.py

```diff
@@ -17,6 +17,7 @@ class Status(models.TextChoices):
         VALID = "valid"
         ERRORING = "erroring"
         DELETING = "deleting"
+        TIMED_OUT = "timed_out"
```

Comment:
Is there a way to get back out of the timed out state if you add the DNS states at a later date?